### PR TITLE
No keys questions

### PR DESCRIPTION
**Pull request by limone-eth**

### CODE REVIEW

File: frontend/app/(authenticated)/question/[id]/page.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function QuestionPage() {
     refetch();
   };
 
-  const isOpenQuestion = !question?.replierId;
+  const isOpenQuestion = !question?.replierId || !profile.hasLaunchedKeys;
```
Comment (0xForkHunter) [inline frontend/app/(authenticated)/question/[id]/page.tsx:35]: I believe this condition is incorrect. If user hasn't launched his keys, it doesn't make it an open-question.
An open-question is special kind of question that doesn't have a target user. Any user can reply to it

File: frontend/components/app/[wallet]/questions-list.tsx [hunk 2]
```diff
@@ -30,6 +30,8 @@ export const QuestionsList: FC<Props> = ({ onBuyKeyClick, type, profile }) => {
     return <LoadingPage />;
   }
 
+  profile.hasLaunchedKeys = false;
```
Comment (0xForkHunter) [inline frontend/components/app/[wallet]/questions-list.tsx:33]: Is this a leftover from testing ?

File: frontend/app/(authenticated)/question/[id]/page.tsx (hunk 1 above)
Comment (limone-eth) [reply frontend/app/(authenticated)/question/[id]/page.tsx:35]: Agreed, but do you think we should still show the comment list below in that component if profile didn't launch a key?

Comment (0xForkHunter) [reply frontend/app/(authenticated)/question/[id]/page.tsx]: I'm not sure I understand you question, but:

Comment list should always be visible for open questions.
The content of the comment itself is removed by the backend if the user doesn't have a key.

For regular questions, comments should be visible if the question has been replied to.
If the question is not replied, it's not possible to add any comment.

Comment (limone-eth) [reply frontend/app/(authenticated)/question/[id]/page.tsx]: "The content of the comment itself is removed by the backend if the user doesn't have a key."

But now we may want to allow showing the comment if the receiving user didn't launch a key I guess

File: frontend/backend/user/user.ts [hunk 3]
```diff
@@ -103,7 +103,18 @@ export const getUser = async (wallet: string) => {
     }
   });
 
-  if (!res) return { error: ERRORS.USER_NOT_FOUND };
+  if (!res) {
```
Comment (limone-eth) [inline frontend/backend/user/user.ts:106]: @francisco-leal @0xForkHunter for some reason i can't perform an OR query here (lint errors), can you help?

**Pull request merged**